codex-rs/tui/src/app.rs

```diff
@@ -36,6 +53,21 @@ enum AppState<'a> {
     GitWarning { screen: GitWarningScreen },
 }
 
+/// State for tracking a pending summarization request
```

Comment:
Again, the previous order was more appropriate: `App` is the most important thing and should be listed before these structs.